components/telegram/sources/new-updates/new-updates.js

```diff
@@ -0,0 +1,54 @@
+const telegram = require("../../telegram.app.js");
+
+module.exports = {
+  key: "telegram-new-updates",
+  name: "New Updates (Instant)",
+  description: "Emits an event for each new Telegram event.",
+  version: "0.0.1",
+  props: {
+    db: "$.service.db",
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+    telegram,
+    updateTypes: { propDefinition: [telegram, "updateTypes"] },
+  },
+  hooks: {
+    async activate() {
+      const response = await this.telegram.createHook(this.http.endpoint, this.updateTypes);
+      console.log(response.data.description);
```

Comment:
same as above, need to save the hook id you get back and retrieve it in the deactivate hook and pass to the delete method